jar/jar.go

```diff
@@ -28,15 +28,11 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/dustin/go-humanize"
```

Comment:
Can we drop this dependency? :) The error message probably isn't worth importing an entire project that we need to manage.

Reply:
Done